core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java

```diff
@@ -74,6 +75,12 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   }
 
   @Override
+  protected boolean shouldSkipCreateAtConfigure() {
+    return tableSinkConfig.containsMacro(Properties.Table.PROPERTY_SCHEMA) ||
```

Comment:
How does these two values being macros affect the creation of the dataset during configure time? Doesn't it only affect the way StructuredRecord->Put transform happens?